src/Item/Item.model.ts

```diff
@@ -190,10 +204,29 @@ export const ItemQueries = Object.freeze({
           SELECT DISTINCT ON (items.id) items.id, items.*
             FROM ${raw(Item.tableName)} items
               ${
-                status
-                  ? SQL`JOIN ${raw(
-                      ItemCuration.tableName
-                    )} item_curations ON items.id = item_curations.item_id`
+                status || synced !== undefined
```

Comment:
Question, isn't a bit weird that we're receiving the `status` and seemingly not using it? I mean it's used to add a join, but the status value is ignored right?

The concern arise because reading this:
```ts
    const query = ItemQueries.selectByCollectionIds(
      collectionIds,
      undefined,
      synced,
      limit,
      offset
    )
```

was a bit confusing (having `undefined` in the middle like that) but reading the code, it'd been the same if you supplied any Curation status or truthy value, right?